src/components/standalone/users_database/CreateOrEditUserDrawer.vue

```diff
@@ -0,0 +1,221 @@
+<script setup lang="ts">
+import {
+  MessageBag,
+  validatePassword,
+  validateRequired,
+  validateStringEqual
+} from '@/lib/validation'
+import { ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import {
+  NeButton,
+  NeTextInput,
+  NeSideDrawer,
+  NeInlineNotification,
+  getAxiosErrorMessage
+} from '@nethserver/vue-tailwind-lib'
+import { ValidationError, ubusCall } from '@/lib/standalone/ubus'
+import type { User } from './UsersDatabaseManager.vue'
+
+const props = defineProps<{
+  isShown: boolean
+  itemToEdit: User | null
```

Comment:
You can use this
```suggestion
  itemToEdit?: User
```
and even forget to set it every time